job_templates/regenerate_puppet_certificate.erb

```diff
@@ -0,0 +1,51 @@
+<%#
+name: Regenerate puppet certificate
+job_category: Miscellaneous
+snippet: false
+template_inputs:
+- name: Delete certificate on Puppet CA
+  required: true
+  input_type: user
+  description: If yes is selected, Foreman will contact Puppet CA of a host and remove
+    the certificate from there. However this requires safe mode to be disabled.
+  options: "no\r\nyes"
+  advanced: false
+provider_type: SSH
+kind: job_template
+%>
+
+<%#
```

Comment:
Is this duplicated metadata section intentional?

Reply:
nope, dropped